db/migrate/20161031202515_create_responses.rb

```diff
@@ -0,0 +1,12 @@
+class CreateResponses < ActiveRecord::Migration
+  def change
+    create_table :responses do |t|
+      t.string :text
+      t.datetime :created_at
```

Comment:
`:created_at` and `updated_at` is provided by `t.timestamps`. Schema looks find but you can leave these out of a migration next time